app/src/main/java/com/github/se/eduverse/ui/camera/PicTakenScreen.kt

```diff
@@ -19,23 +19,35 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asAndroidBitmap
 import androidx.compose.ui.graphics.asImageBitmap
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.unit.dp
 import com.github.se.eduverse.R
+import com.github.se.eduverse.model.Photo
 import com.github.se.eduverse.ui.navigation.NavigationActions
+import com.github.se.eduverse.viewmodel.PhotoViewModel
+import java.io.ByteArrayOutputStream
 import java.io.File
 
 fun adjustImageRotation(bitmap: Bitmap): Bitmap {
   val matrix = Matrix()
-  matrix.postRotate(90f) // Ajuste cette rotation si nécessaire
+  matrix.postRotate(90f)
   return Bitmap.createBitmap(bitmap, 0, 0, bitmap.width, bitmap.height, matrix, true)
 }
 
 @Composable
-fun PicTakenScreen(photoFile: File?, navigationActions: NavigationActions) {
+fun PicTakenScreen(
+    photoFile: File?,
+    navigationActions: NavigationActions,
+    viewModel: PhotoViewModel
+) {
+  val ownerId = "user123"
```

Comment:
I feel like you should include a FirebaseAuth to the arguments of the function and use auth.currentUser.uid here